common/src/main/java/io/netty/util/internal/ExceptionUtils.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2012 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.util.internal;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public final class ExceptionUtils {
+    /**
+     * <p>Gets the stack trace from a Throwable as a String.</p>
+     *
+     * This method has been cherry-picked from {@link org.apache.commons.lang3.exception.ExceptionUtils}
+     *
+     * @param throwable  the <code>Throwable</code> to be examined
+     * @return the stack trace as generated by the exception's
+     *  <code>printStackTrace(PrintWriter)</code> method
+     */
+    public static String getStackTrace(final Throwable throwable) {
```

Comment:
Please replace this method with this that can be found here:
https://github.com/netty/netty/blob/4.1/transport/src/main/java/io/netty/channel/ChannelOutboundBuffer.java#L700

And also reuse it there then :) 

With this you can also remove the link to lang3.